pkg/controller/machine/controller.go

```diff
@@ -238,3 +246,16 @@ func (r *ReconcileMachine) isDeleteAllowed(machine *clusterv1.Machine) bool {
 	// own machine.
 	return node.UID != machine.Status.NodeRef.UID
 }
+
+func (r *ReconcileMachine) deleteNode(context context.Context, name string) error {
```

Comment:
Nit: `s/context/ctx` so you don't override `context` package identifier

Reply:
> Nit: `s/context context.Context/ctx context.Context/` so you don't override `context` package identifier

And using `ctx` seems to be the common form.